yaml/csr.yaml.m4

```diff
@@ -4180,6 +4180,10 @@ ifelse(eval(UMODE != 0), 1, [[[
         instraccessfault:     1
         illegalinstr:         2
         breakpoint:           3
+ifelse(eval(UMODE == 0), 1, [[[
```

Comment:
Using UMODE does not seem to make sense here; should be fixed on both 40X and 40S later.